blueprints/scaffold/index.js

```diff
@@ -72,6 +83,24 @@ module.exports = {
       this._writeRouterStatus(status, 'red');
     }
   },
+  _mirageVersionIsOld: function(options) {
+    var basePath = options.target;
+    var oldPath = '/app/mirage/config.js';
+    var newPath = '/mirage/config.js';
+    try {
+      fs.statSync(basePath + newPath);
+      return false;
+    }
+    catch(_) {
+    }
+    try {
+      fs.statSync(basePath + oldPath);
+      return true;
+    }
+    catch(e) {
+      throw new Error("Can't tell what version of Mirage this is: can't find config file. Tried " + newPath + " and " + oldPath + ". Got " + e);
+    }
```

Comment:
What about to check the ember-cli-mirage version directly from package.json, instead of feature detection?